src/ragas/metrics/_domain_specific_rubrics.py

```diff
@@ -158,6 +164,12 @@ async def _multi_turn_ascore(
         prompt_input = MultiTurnInputWithoutRubric(
             user_input=interaction,
         )
+
+        rubrics_text = "\n".join(
+            f"{key}: {value}" for key, value in self.rubrics.items()
+        )
+        self.multi_turn_scoring_prompt.instruction = f"{self.multi_turn_scoring_prompt.instruction}\n\nScoring Rubrics:\n{rubrics_text}\n"
```

Comment:
self.multi_turn_scoring_prompt.instruction += f"\n\nScoring Rubrics:\n{rubrics_text}\n"
```suggestion
        self.multi_turn_scoring_prompt.instruction = f"{self.multi_turn_scoring_prompt.instruction}\n\nScoring Rubrics:\n{rubrics_text}\n"
```